src/herder/TxSetFrame.cpp

```diff
@@ -161,14 +190,13 @@ TxSetFrame::surgePricingFilter(Application& app)
 {
     LedgerTxn ltx(app.getLedgerTxnRoot());
     auto header = ltx.loadHeader();
-    size_t max = header.current().maxTxSetSize;
-    if (mTransactions.size() > max)
-    { // surge pricing in effect!
+
+    size_t maxSize = header.current().maxTxSetSize;
```

Comment:
I standardized on `size_t` as it's returned by calls like `mTransactions.size()`.
While it's implementation defined, as we require 64 bit (or better) arch, it will fit 32 bit values easily and we avoid casting all over the place.